unittests/test_utility.py

```diff
@@ -23,22 +23,22 @@
 class TestOSTools(unittest.TestCase):
     def test_command_success(self):
         completed = os_ext.run_command('echo foobar')
-        self.assertEqual(completed.returncode, 0)
-        self.assertEqual(completed.stdout, 'foobar\n')
+        assert completed.returncode == 0
+        assert completed.stdout == 'foobar\n'
 
     def test_command_error(self):
-        self.assertRaises(SpawnedProcessError, os_ext.run_command,
-                          'false', 'check=True')
+        with pytest.raises(SpawnedProcessError):
+            os_ext.run_command('false', 'check=True')
```

Comment:
That test was wrong in the past:

```suggestion
            os_ext.run_command('false', check=True)
```